tools/rabbitmq-quorum-migration.sh

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/bash
+
+RABBITMQ_SERVICES_TO_RESTART=barbican,blazar,cinder,cloudkitty,designate,heat,ironic,keystone,magnum,manila,neutron,nova,octavia
+RABBITMQ_CONTAINER_NAME=rabbitmq
+
+if [[ ! $KAYOBE_CONFIG_PATH ]]; then
+    echo "Environment variable \$KAYOBE_CONFIG_PATH is not defined"
+    echo "Ensure your environment is set up to run kayobe commands"
+    exit 2
+fi
+
+if [[ ! "$1" = "--skip-checks" ]]; then
+    kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/rabbitmq-migration-prechecks.yml -e services_to_restart=$RABBITMQ_SERVICES_TO_RESTART -e rabbitmq_container_name=$RABBITMQ_CONTAINER_NAME
+fi
+
+kayobe overcloud service configuration generate --node-config-dir /etc/kolla --kolla-skip-tags rabbitmq-ha-precheck &&
+kayobe kolla ansible run "stop --yes-i-really-really-mean-it" -kt $RABBITMQ_SERVICES_TO_RESTART &&
+kayobe kolla ansible run rabbitmq-reset-state
+if [[ ! "$1" = "--skip-checks" ]]; then
+    kayobe overcloud host command run -l controllers -b --command "(docker exec $RABBITMQ_CONTAINER_NAME rabbitmqctl list_queues name --silent && docker exec $RABBITMQ_CONTAINER_NAME rabbitmqctl list_exchanges name --silent) | grep -v '^$' | (! grep -v 'amq.')"
```

Comment:
Could be another playbook?

Reply:
I've gone 100% scripty now to keep things cleaner